### ISSUE TITLE
Setting to disable the in-chat button to open the picker [Suggestion]

### ISSUE THREAD

**Issue by DeCat4** [TheLazySquid/DiscordImageFolder#15]
Would be nice to have a setting for the plugin so that it doesn't add the button that opens the images, too much clutter for some people, i'm okay just clicking the GIF button then clicking images.

**Comment by TheLazySquid** [comment]
Good idea! This should now be added, download the new version from [this repo](https://github.com/TheLazySquid/DiscordImageFolder/blob/main/build/ImageFolder.plugin.js) since it hasn't been accepted to the official repo.
